projects/manager/src/styles.scss

```diff
@@ -27,3 +27,12 @@ body {
 a {
     color: mat.get-color-from-palette(map-get($geoengine-theme, foreground), secondary-text);
 }
+
```

Comment:
if I login and it fails (500), I get no error

Reply:
ff2f1bba4b8b75d2092c06f95e03637da2fdf1a6